### PR TITLE
support for sub-profile views, config elements

### PR DESCRIPTION
**Pull request by Erin-Mounts**

### CODE REVIEW

File: mPower2/mPower2/Profile/ProfileTableViewCell.swift [hunk 1]
```diff
@@ -36,9 +36,10 @@ import Research
 
 class ProfileTableViewCell: UITableViewCell {
     
+    @IBOutlet weak var icon: UIImageView?
```
Comment (syoung-smallwisdom) [inline mPower2/mPower2/Profile/ProfileTableViewCell.swift]: You added an outlet, but I don't see it being used?

File: mPower2/mPower2/ProfileTableViewController.swift [hunk 2]
```diff
@@ -40,17 +40,36 @@ class ProfileTableViewController: UITableViewController, RSDTaskViewControllerDe
...
-    static let healthProfileViewControllerSegue = "HealthProfileViewControllerSegue"
-    static let profileItemEditViewControllerSegue = "ProfileItemEditViewControllerSegue"
-    
-    fileprivate lazy var _profileDataSource: SBAProfileDataSource = {
-        return SBAProfileDataSourceObject.shared
-    }()
+//    static let webViewControllerSegueId = "WebViewControllerSegue"
+//    static let withdrawalViewControllerSegueId = "WithdrawalViewControllerSegue"
+//    static let healthProfileViewControllerSegueId = "HealthProfileViewControllerSegue"
+//    static let profileItemEditViewControllerSegueId = "ProfileItemEditViewControllerSegue"
```
Comment (syoung-smallwisdom) [inline mPower2/mPower2/ProfileTableViewController.swift]: TODO comment or remove

File: mPower2/mPower2/Profile/ProfileTableViewCell.swift (hunk 1 above)
Comment (Erin-Mounts) [reply mPower2/mPower2/Profile/ProfileTableViewCell.swift]: Aah thank you! I was going to look into why that wasn't working this morning.

File: mPower2/mPower2/ProfileTableViewController.swift (hunk 2 above)
Comment (Erin-Mounts) [reply mPower2/mPower2/ProfileTableViewController.swift]: 👍

**Pull request merged**